packages/hosted-form-v2/src/iframe-content/hosted-input-manual-order-payment-handler.ts

```diff
@@ -75,12 +75,17 @@ export default class HostedInputManualOrderPaymentHandler {
                 });
             }
         } catch (error) {
-            this._eventPoster.post({
-                type: HostedInputEventType.SubmitManualOrderFailed,
-                payload: this._isPaymentErrorResponse(error)
-                    ? { error: error.body.errors[0], response: error }
-                    : { error: { code: snakeCase(error.name), message: error.message } },
-            });
+            if (this._isPaymentErrorResponse(error)) {
+                this._eventPoster.post({
+                    type: HostedInputEventType.SubmitManualOrderFailed,
+                    payload: { error: error.body.errors[0], response: error },
+                });
+            } else if (this._isErrorResponse(error)) {
+                this._eventPoster.post({
+                    type: HostedInputEventType.SubmitManualOrderFailed,
+                    payload: { error: { code: snakeCase(error.name), message: error.message } },
+                });
+            }
```

Comment:
Should error be thrown if it is not `ErrorResponse` or `PaymentErrorResponse`?

Reply:
We didn't throw error previously. And `this._isErrorResponse(error)` should catch the rest cases.